### DISCUSSION TITLE
关于5.2更新中英文公式识别能力的疑惑

### DISCUSSION THREAD

**Discussion by ManiiXu** [OleehyO/TexTeller#48] [Q&A] [answered]
想问下[2024-05-02] Support mixed Chinese English formula recognition.具体是指什么样的功能或识别能力。测试如果是公式里带有中文字符并不能正常识别出来，看HF模型的tokenizer_config.json、vocab.jsont也没有更新中文的词。

**Comment by OleehyO** [comment, marked as answer]
具体来说就是可以把中英文文字和公式混在一起识别（需要在web demo的侧栏里打开混合识别的选项），例如这张图：
<img width="576" alt="截屏2024-05-06 12 40 36" src="https://github.com/OleehyO/TexTeller/assets/56267907/617bb1f1-fa50-4044-8235-339032fbcfff">

混合识别会进行文字OCR+公式OCR，所以公式识别模型的tokenizer并不需要有中文的token。

**Reply by OleehyO** [reply]
为了防止概念混淆，新版本已经把中英文公式识别更名为段落识别

**Comment by OleehyO** [comment]
使用混合识别后推理速度会很慢，而且目前的版本效果还没有做的很好，等新一版的公式识别模型训练好以后（大概6月初）应该会改善很多。
